practica-4.html

```diff
@@ -378,6 +378,34 @@ <h5 class="card-title" >Goliza del madrid al atletico</h5>
           </div>
         </div> <!--End row-->
       </div> <!--End container-->
+      <!--Footer-->
+      <div class="container-fliud text-center pb-5 pt-2 px-3" style="background-color: #e8eaed;">
```

Comment:
It looks good, just give it a little margin in the top

Reply:
okay ill put it